tests/codeception/frontend/functional/SignupCest.php

```diff
@@ -10,29 +10,27 @@ class SignupCest
 
     /**
      * This method is called before each cest class test method
-     * @param \Codeception\Event\TestEvent $event
+     * @param \codeception_frontend\FunctionalTester $I
      */
-    public function _before($event)
+    public function _before($I)
     {
     }
 
     /**
      * This method is called after each cest class test method, even if test failed.
-     * @param \Codeception\Event\TestEvent $event
+     * @param \codeception_frontend\FunctionalTester $I
      */
-    public function _after($event)
+    public function _after($I)
     {
-        User::deleteAll([
-            'email' => 'tester.email@example.com',
-            'username' => 'tester',
-        ]);
+        //reload default fixtures
+        $I->loadFixtures();
```

Comment:
No, it shouldn't because `unloadFixtures` [does nothing with a DB](https://github.com/yiisoft/yii2/blob/master/framework/test/BaseActiveFixture.php#L112).